tests/testJointcal.py

```diff
@@ -0,0 +1,239 @@
+
+        # position of the Twinkles run 1 catalog
+        center = coord.IcrsCoord(53.00914*geom.degrees, -27.43895*geom.degrees)
+        r = geom.Angle(3, geom.degrees)
+        config = astrom.LoadAstrometryNetObjectsTask.ConfigClass()
+        refLoader = astrom.LoadAstrometryNetObjectsTask(config=config)
+        reference = refLoader.loadSkyCircle(center, r, filterName='r')
+        self.reference = reference.refCat.copy()  # for in-memory contiguity.
+
+        # Get a set of source catalogs.
+        self.catalogs = []
+        butler = persistence.Butler(os.path.join(data_dir, 'twinkles1'))
+        for visit in butler.queryMetadata('src', 'visit'):
+            src = butler.get('src', dataId={'visit': visit})
+            dataId = butler.dataRef('src', visit=visit).dataId
+            calexp_md = butler.get('calexp_md', {'visit': visit, 'raft': '2,2', 'sensor': '1,1'})
+            self.catalogs.append(FakeRef(src, calexp_md, dataId))
+
+        self.jointcalTask = jointcal.JointcalTask()
+
+    @unittest.skip('jointcal currently fails if only given one catalog!')
+    def testJointCalTask_1_catalog(self):
+        self.jointcalTask.run(self.catalogs[:1])
+
+    def testJointCalTask_2_catalog(self):
+        ncat = 2
+        self.jointcalTask.run(self.catalogs[:ncat])
+        rms_rel, rms_abs = compute_rms(self.reference, self.catalogs[:ncat], fluxlimit=100)
+        # NOTE: this number come from an initial jointcal run.
+        self.assertLess(rms_rel, 8.4e-3/206265)
+        self.assertLess(rms_abs, absolute_error)
+
+    @unittest.skip('Keeping this around for diagnostics on the behavior with n catalogs.')
+    def testJointCalTask_4_catalog(self):
+        ncat = 4
+        self.jointcalTask.run(self.catalogs[:ncat])
+        rms_rel, rms_abs = compute_rms(self.reference, self.catalogs[:ncat], fluxlimit=100)
+        # NOTE: this number come from an initial jointcal run.
+        self.assertLess(rms_rel, 7.8e-3/206265)
+        self.assertLess(rms_abs, absolute_error)
+
+    @unittest.skip('Keeping this around for diagnostics on the behavior with n catalogs.')
+    def testJointCalTask_7_catalog(self):
+        ncat = 7
+        self.jointcalTask.run(self.catalogs[:ncat])
+        rms_rel, rms_abs = compute_rms(self.reference, self.catalogs[:ncat], fluxlimit=100)
+        # NOTE: this number come from an initial jointcal run.
+        self.assertLess(rms_rel, 7.5e-3/206265)
+        self.assertLess(rms_abs, absolute_error)
+
+    def testJointCalTask_10_catalog(self):
+        ncat = 10
+        self.jointcalTask.run(self.catalogs[:ncat])
+        rms_rel, rms_abs = compute_rms(self.reference, self.catalogs[:ncat], fluxlimit=100)
+        # NOTE: this number comes from an initial jointcal run.
+        self.assertLess(rms_rel, 7.4e-3/206265)
+        self.assertLess(rms_abs, absolute_error)
+
+
+def rms_per_star(data):
+    """Compute the RMS/catalog star; data is a dict of lists of distances."""
```

Comment:
The result appears to be a list of lists of distances. I don't see a dict being made. Am I missing something?